gen/definitions/generic/service_lan_vpn_ethernet_interface_feature_associate_dhcp_server.yaml

```diff
@@ -0,0 +1,300 @@
+---
+name: Service LAN VPN Interface Ethernet Feature Associate DHCP Server Feature
```

Comment:
Shouldn't we add something similar for below two resources that also can have DHCP server associated?
- sdwan_service_lan_vpn_interface_ipsec_feature
- sdwan_service_lan_vpn_interface_svi_feature